README.md

```diff
@@ -202,15 +202,44 @@ let twimlParamTo = "to"
 
 Use the text field to specify the identity of the call receiver, then tap the “Call” button to make a call. The TwiML parameters used in `TwilioVoice.connect()` method should match the name used in the server.
 
-<kbd><img width="500px" src="https://github.com/twilio/voice-quickstart-swift/raw/master/Images/client-to-client.png"/></kbd>
+<kbd><img width="500px" src="https://github.com/twilio/voice-quickstart-ios/raw/master/Images/client-to-client.png"/></kbd>
 
 ### <a name="bullet12"></a>12. Make client to PSTN call
 A verified phone number is one that you can use as your Caller ID when making outbound calls with Twilio. This number has not been ported into Twilio and you do not pay Twilio for this phone number.
 
 To make client to number calls, first get a valid Twilio number to your account via https://www.twilio.com/console/phone-numbers/verified. Update your server code and replace the caller number variable  (`CALLER_NUMBER` or `callerNumber` depending on which server you chose) with the verified number. Restart the server so it uses the new value. Voice Request URL of your TwiML application should point to the public URL of your application server’s `/makeCall` endpoint.
 
-<kbd><img width="500px" src="https://github.com/twilio/voice-quickstart-swift/raw/master/Images/client-to-pstn.png"/></kbd>
-
+<kbd><img width="500px" src="https://github.com/twilio/voice-quickstart-ios/raw/master/Images/client-to-pstn.png"/></kbd>
+
+## <a name="custom-audio-device-example"></a> Custom Audio Device Example
+
+The `ExampleAVAudioEngineDevice` uses CoreAudio's VoiceProcessingIO audio unit to playback and record audio at up to 48KHz with built-in echo cancellation. The example uses two AVAudioEngine in manual rendering mode:
+
+1. The upstream AVAudioEngine is used for mixing the Remote Participant's audio with audio from a file. The AudioUnit receives mixed audio samples from AVAudioEngine's output node.
```

Comment:
How much effort would it be to do this https://developer.apple.com/documentation/accelerate/equalizing_audio_with_vdsp instead?

Reply:
I think the custom audio device that mixes music is a more practical use case than an example that renders audio output visually.

But maybe you were thinking this could be an audio energy or power indicator?

Reply:
If I understood the current implementation, it's mixing the output? I can imagine mixing audio with input is more practical than mixing with the output?

Reply:
It's both, it gets mixed into what is played out locally (music + remote participant audio) and mixes it with what is being sent (music + local participant audio/microphone)

Reply:
Got it. That makes sense. Sounds good